### PR TITLE
Type annotations and map fixes

### PR DESCRIPTION
**Pull request by estellecomment**

### CODE REVIEW

File: matrix-js-sdk/src/models/event-timeline-set.js [hunk 1]
```diff
@@ -80,13 +80,21 @@ function EventTimelineSet(room, opts) {
 
     // just a list - *not* ordered.
     this._timelines = [this._liveTimeline];
+    /**
+     * Map event type → event timeline
```
Comment (weeman1337) [inline matrix-js-sdk/src/models/event-timeline-set.js]:
```suggestion
     * Map event id → event timeline
```

File: matrix-js-sdk/src/models/relations.js [hunk 2]
```diff
@@ -41,7 +41,16 @@ export default class Relations extends EventEmitter {
         this.relationType = relationType;
         this.eventType = eventType;
         this._relations = new Set();
+        /**
+         * Map relation -> set of events
+         * not sure about types, but probably :
+         * @type Map<Object, Set<Event>>
```
Comment (weeman1337) [inline matrix-js-sdk/src/models/relations.js:47]: I think that is the best we can do at the moment.
`Object` is a relation in this case.

**Review by weeman1337 — APPROVED** [review]
Ok :+1:  just some small comments.

**Comment by mcalinghee** [comment]
Looks good

**Review by mcalinghee — APPROVED** [review]

**Pull request merged**